app/forms/sufia/forms/collection_form.rb

```diff
@@ -1,6 +1,4 @@
 module Sufia::Forms
```

Comment:
This class now has a method, so it is still in the code, but I did remove app/models/concerns/sufia/collection_behavior.rb which is empty.